src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -95,6 +96,7 @@
 import com.wikia.webdriver.pageobjectsfactory.pageobject.special.preferences.EditingPreferencesPageObject;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.special.preferences.PreferencesPageObject;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.special.watch.WatchPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.template.TemplatePageObject;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.template.TemplatePageObject'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)